query-graphs-js/src/pathTree.ts

```diff
@@ -384,7 +398,7 @@ export class PathTree<TTrigger, RV extends Vertex = Vertex, TNullEdge extends nu
     if (iterResult.done) {
       return this;
     }
-    const [edge, trigger, conditions] = iterResult.value;
+    const [edge, trigger, conditions, contextToSelection, parameterToContext] = iterResult.value;
```

Comment:
You can delete the method this code is a part of (`mergePathInternal()`) and its only caller (`mergePath()`), as they're both dead code. You can also delete `childsFromPathElements()`, as its also only called by this method.

Reply:
I love deleting dead code :)